src/ripple/app/ledger/InboundLedger.h

```diff
@@ -160,6 +162,8 @@ class InboundLedger
 
     std::set <uint256> mRecentNodes;
 
+    SHAMapAddNode      mStats;
```

Comment:
I don't see a problem with two styles in the same file. In fact, I prefer it. It makes it easy to see which parts of the code are old and haven't been updated. As this pull request stands, I would use `stats_` for the new member and leave the existing member names alone. In the future if I was going to do a deeper refactoring and tidying of the class then only at that point would I rename all the members to the newer style.